### PR TITLE
ac_bee OK

### PR DESCRIPTION
**Pull request by inspectredc**
Mostly documented but couldnt get some function names from player lib. The main action function seems fairly different from AC (which has 4 action functions rather than 3). 

Also added in neater inclusion of data in ac_fuusen that I realised after doing a similar function here

### CODE REVIEW

File: include/audio.h [hunk 1]
```diff
@@ -32,7 +32,7 @@ void sAdo_MessageSpeed(u8 speed);
 u8 sAdo_MessageSpeedGet(void);
 void sAdo_SysLevStart(u8 id);
 void sAdo_SysLevStop(u8 id);
-void sAdo_OngenPos(s32 arg0, u8 arg1, struct xyz_t* pos);
+void sAdo_OngenPos(uintptr_t arg0, u8 arg1, struct xyz_t* pos);
```
[the remark this answers is not in the document]
Comment (inspectredc) [reply include/audio.h:35]: ah ok, i couldnt find other uses which didnt use ptrs but if its just a unique identifier which doesn't necessarily use pointers then that makes sense

File: src/overlays/actors/ovl_Fuusen/ac_fuusen.c [hunk 2]
```diff
@@ -41,10 +41,9 @@ ActorProfile Fuusen_Profile = {
 };
 
 void aFSN_actor_ct(Actor* thisx, Game_Play* game_play) {
-    static xyz_t Init_Size = { 0.01f, 0.01f, 0.01f };
     Fuusen* this = THIS;
     SkeletonInfoR* skeletonInfo = &this->skeletonInfo;
-    xyz_t size = Init_Size;
+    xyz_t Init_Size = { 0.01f, 0.01f, 0.01f };
```
Comment (emilybrooks) [inline src/overlays/actors/ovl_Fuusen/ac_fuusen.c:46]: I'm confused by this but I think it should keep the static keyword?

Comment (inspectredc) [reply src/overlays/actors/ovl_Fuusen/ac_fuusen.c:46]: it wont match if it does

Comment (inspectredc) [reply src/overlays/actors/ovl_Fuusen/ac_fuusen.c:46]: I thought it made more sense for them to write it like this than what I previously put, but only realised when I had the same issue here in bee

**Review by emilybrooks — APPROVED** [review]

**Pull request merged**